client/post-editor/editor-publish-button/index.jsx

```diff
@@ -87,8 +88,16 @@ export class EditorPublishButton extends Component {
 			case 'update':
 				return this.props.translate( 'Update' );
 			case 'schedule':
+				if ( config.isEnabled( 'post-editor/delta-post-publish-flow' ) ) {
+					return this.props.translate( 'Schedule…',
```

Comment:
Hi! I've found a possible matching string that has already been translated **75** times:
`translate( 'Schedule' )` *ES Score: 21*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor-sidebar-indication)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).